server/db/db.js

```diff
@@ -0,0 +1,51 @@
+const { open } = require('sqlite');
+const sqlite3 = require('sqlite3').verbose();
+let db;
+
+ (async () => {
```

Comment:
While this works, it looks a little sloppy.  Why don't you make an async function called `initialize` and then execute it immediately after?